### ISSUE TITLE
Launch Plan: The PostHog Pyramid Scheme

### ISSUE THREAD

**Issue by joethreepwood** [PostHog/meta#205]
Let's finish the referral program feature set we did at the hackathon. 

## Best practices
- [x] Ensure the product has at least one customer story within 3 weeks of launch (@joethreepwood will write about our usage) - https://github.com/PostHog/posthog.com/pull/8462
- [x] Ensure we publish best practice content for the product and link to it from docs (@joethreepwood will cover this) - https://github.com/PostHog/posthog.com/pull/8462
- [ ] Ensure that products have at least one pre-made template (or similar) for users (@raquelmsmith unsure if we need this?)
- [x] Ensure the product has at least one tutorial at launch (@joethreepwood will cover this in the marketing PR) - https://github.com/PostHog/posthog.com/pull/8462
- [x] Ensure the product has a robust docs page, and product page if needed - https://github.com/PostHog/posthog.com/pull/8462

## Development to-dos
The following items have to be completed before launch, taken from the Pyramid Scheme channel on Slack. 

- [x] Finish up the referrer page
- [ ] Add the ability to add/edit referrers and redeemers
- [ ] Make referral links work when directing to posthog.com (not us or eu instances)
- [ ] Sort out the event sending
- [ ] Trigger the tracking event on sign-up
- [ ] Trigger tracking event somehow when someone pays their first bill

Stretch goals for the future:
- [ ] Make API to get all redeemers for a given referrer for any program in the current team
- [ ] Track referral attribution with posthog-js (not in a custom cookie)
- [ ] Manually create referrer
- [ ] Verify referral code (via search)
- [ ] Referral code customization
- [ ] Redeem the code via posthog-js
- [ ] Program analytics (referrals over time, shares, etc)
- [ ] Embeddable components for referral widgets 
- [ ] Link sharing attribution
- [ ] Shopify integration
- [ ] Stripe integration

## Launch plan
Below will all be done by @joethreepwood, unless otherwise stated.

- [x] Key art by @lottiecoxon - https://github.com/PostHog/posthog.com/issues/8476
- [ ] Announce on changelog
- [ ] Announce in changelog newsletter
- [ ] Announce in PostHog for Startups newsletter
- [ ] Announce with Pylon broadcast to all users
- [ ] Add to Product OS page - https://github.com/PostHog/posthog.com/pull/8462
- [ ] Add to onboarding email flow

All feedback and additional ideas from @PostHog/marketing is appreciated.
